tsconfig.json

```diff
@@ -11,7 +11,7 @@
     // "disableReferencedProjectLoad": true,             /* Reduce the number of projects loaded automatically by TypeScript. */
 
     /* Language and Environment */
-    "target": "ES2022",                                  /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */
+    "target": "ES2022" /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */,
```

Comment:
What is this style called? :scream:

Reply:
Ah, funny.  It's using the Prettier style guide. However, I wasn't aware of how it would format JSON with comments. It doesn't really appear beautiful.